src/crypto.rs

```diff
@@ -593,9 +593,12 @@ impl PrivateKey {
         der_key: &[u8],
         scheme: SignatureScheme,
     ) -> Result<Self> {
-        let key_pair =
-            EcdsaKeyPair::from_pkcs8(&ECDSA_P256_SHA256_ASN1_SIGNING, der_key)
-                .unwrap();
+        let key_pair = EcdsaKeyPair::from_pkcs8(
+            &ECDSA_P256_SHA256_ASN1_SIGNING,
+            der_key,
+            &SystemRandom::new(),
```

Comment:
can you clarify this change in the PR desc?

Reply:
Done

Reply:
Did the API change to require passing in an RNG? Is this a cryptographically safe RNG to use here?

Reply:
Yes. It is also cryptographically safe; the same API generates new keys and sign objects.

https://docs.rs/ring/latest/ring/rand/struct.SystemRandom.html